.vib/rabbitmq/cypress/cypress/integration/rabbitmq_spec.js

```diff
@@ -0,0 +1,100 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows login/logout', () => {
+  const APP_NAME = 'RabbitMQ';
+
+  cy.login();
+  cy.get('#versions').should('contain', APP_NAME);
+  cy.contains('.hider', 'Queued messages');
```

Comment:
The `logout` test failed intermittently since the button would get clicked before the event listener listened.
I handled this in two ways:

- This assertion, will make sure the page will load before searching and clicking on the Log out button
- Increase click timeout by 200 ms (1100)

Reply:
I've improved the approach here, reducing the click timeout. 
No need to intercept the call, but I'll keep this for future reference.